### ISSUE TITLE
Request for new release

### ISSUE THREAD

**Issue by tmplt** [HalosGhost/pbpst#94]
I was trying to package the latest release for NixOS but stumbled upon some portability issues with tup, which has since then been removed as a build dependency. Packaging of master works, however; could a new release be made?

**Comment by HalosGhost** [comment]
Absolutely. And it could be done right away. The only thing holding back the next release would be fixing `stdin` pasting / removing the now-deprecated `curl_form` interface in favor of `curl_mime`. I have been quite busy with house renovations and have not had time/motivation to work much on personal projects, but this project is not abandoned!

Help is definitely accepted (actually, the start of the work has already been contributed over in the PR section), but hopefully I will be able to get back to it if not in the next couple weekends, shortly thereafter.

**Comment by tmplt** [comment]
Thanks for the reply. In the meantime, latest HEAD of master was just packaged and merged in https://github.com/NixOS/nixpkgs/pull/49459.
